packages/terra-application-utility/src/header/_HeaderUtilityMenu.jsx

```diff
@@ -0,0 +1,256 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import classNames from 'classnames/bind';
+import 'terra-base/lib/baseStyles';
+import ContentContainer from 'terra-content-container';
+import IconLeft from 'terra-icon/lib/icon/IconLeft';
+import MenuDivider from '../_UtilityMenuDivider';
+import Utils from '../_Utils';
+import MenuItem from './_HeaderUtilityMenuItem';
+import styles from './_HeaderUtilityMenu.scss';
+
+const cx = classNames.bind(styles);
+
+const propTypes = {
+  /**
+   * Key of the top level menu.
+   */
+  initialSelectedKey: PropTypes.string.isRequired,
+  /**
+   * Indicates if the height is bound to a value.
+   */
+  isHeightBounded: PropTypes.bool,
+  /**
+   * The config file containing the menu items to be rendered.
```

Comment:
The ~config file containing~ **array of** the menu item~s~ configs to be rendered.